src/workloads/docs/InsertWithNop.yml

```diff
@@ -11,16 +11,16 @@ Actors:
   Threads: 100
   Database: "test2"
   Phases:
-  - Operation: Nop
+  - &Nop {Nop: true}
   - Collection: "inserts"
     Document: {"a": {^RandomInt: {min: 50, max: 60}}}
-  - Operation: Nop
-  - Operation: Nop
+  - *Nop
+  - *Nop
   - Collection: "inserts"
     Document: {"b": {^RandomString: {length: {^RandomInt: {min: 5, max: 15}}, alphabet: abcde}}}
   - Collection: "inserts"
     Document: {"a": {^RandomInt: {min: 63, max: 70}}}
-  - Operation: Nop
+  - *Nop
   - Collection: "inserts"
     Document: {"b": {^RandomString: {length: {^RandomInt: {min: 15, max: 25}}, alphabet: abcde}}}
-  - Operation: Nop
+  - *Nop
```

Comment:
Personally I'd rather avoid this rather silly anchor and just put `Nop: true` in these lines